### PR TITLE
Update test-plan.md

### PR DESCRIPTION
**Pull request by ryanbrandenburg**
Include some new Razor scenarios.

CC @NTaylorMullen and @ajaybhargavb 

@JoeRobich here's the reaction I should have included on https://github.com/OmniSharp/omnisharp-vscode/pull/3524.

### CODE REVIEW

File: test-plan.md [hunk 1]
```diff
@@ -106,6 +106,9 @@ To setup a test project to verify on you can do:
 * Completion is available for types that exist in the project (i.e. `Program`)
 * Typing `@model DateTime` prompts for completion for the `model` symbol and the `DateTime` symbol.
 
+#### C# Lightbulbs
```
Comment (ajaybhargavb) [inline test-plan.md]:
```suggestion
##### C# Lightbulbs
```

File: test-plan.md [hunk 2]
```diff
@@ -137,6 +140,9 @@ To setup a test project to verify on you can do:
 ##### C# Diagnostics
 * Typing `@ThisDoesNotExist` results in an error being created and squiggled in the .cshtml file. NOTE: This error squiggly will be misaligned due to known issues.
 
+#### TagHelper Quick Info
```
Comment (ajaybhargavb) [inline test-plan.md]:
```suggestion
##### TagHelper Quick Info
```

File: test-plan.md [hunk 3]
```diff
@@ -106,6 +106,9 @@ To setup a test project to verify on you can do:
 * Completion is available for types that exist in the project (i.e. `Program`)
 * Typing `@model DateTime` prompts for completion for the `model` symbol and the `DateTime` symbol.
 
+#### C# Lightbulbs
+* Typing `@{ var x = new HtmlString("sdf"); }`results in squiggles under `HtmlString`. Selecting `HtmlString` and pressing Ctrl+. results in a change suggestion to fully-qualify that type name as `Microsoft.AspNetCore.Html.HtmlString`. Hitting enter on that suggestion resolves the squiggles and changes the line.
```
Comment (NTaylorMullen) [inline test-plan.md]: You may want to be a little more specific here on which entry to select to future proof ourselves for when we add more light bulbs

File: test-plan.md [hunk 4]
```diff
@@ -137,6 +140,9 @@ To setup a test project to verify on you can do:
 ##### C# Diagnostics
 * Typing `@ThisDoesNotExist` results in an error being created and squiggled in the .cshtml file. NOTE: This error squiggly will be misaligned due to known issues.
 
+#### TagHelper Quick Info
+* Typing `<environment></environment>` then hovering the mouse over the opening tag results in an informative window about that TagHelper.
```
Comment (NTaylorMullen) [inline test-plan.md]: I'd provide an example with a disclaimer that the text may not be exact

File: test-plan.md (hunk 3 above)
Comment (ryanbrandenburg) [reply test-plan.md]: I feel like I was super specific, `Microsoft.AspNetCore.Html.HtmlString` is the entire text which is shown by that light bulb. Can you think of a way I could make that wording clearer?

Comment (NTaylorMullen) [reply test-plan.md]: ya this is why you don't leave comments later at night. You were totally clear enough lol.

File: test-plan.md (hunk 4 above)
Comment (ryanbrandenburg) [reply test-plan.md]:
```suggestion
* Typing `<environment></environment>` then hovering the mouse over the opening tag results in a window about that TagHelper that should read something like `EnvironmentTagHelper ITagHelper implementation targeting <environment> elements that conditionally renders content based on the current value of IHostingEnvironment.EnvironmentName...`.
```

**Comment by ryanbrandenburg** [comment]
Updated with suggestions, @JoeRobich let us know if there's anything else you need from us.

**Comment by JoeRobich** [comment]
@ryanbrandenburg Thanks!

**Pull request merged**